From 20923dca4cf5a2c2559524627022cf89ae608993 Mon Sep 17 00:00:00 2001
From: DcraftBg <czl_obshta@abv.bg>
Date: Fri, 27 Dec 2024 19:05:54 +0200
Subject: [PATCH] MinOS support

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index f93583e..7bb2d0e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
-CC:=gcc
-CFLAGS:=-Wall -Werror -Wextra
+CC:=x86_64-minos-gcc
+CFLAGS:=-Wall -Werror -Wextra -Wno-nonnull
 all: dim
 dim: src/main.c
 	$(CC) $(CFLAGS) $< -o $@
-- 
2.34.1

